lxd/instance/instance_utils.go

```diff
@@ -18,7 +18,7 @@ import (
 	liblxc "github.com/lxc/go-lxc"
 	"github.com/pborman/uuid"
 
-	"github.com/canonical/lxd/client"
+	lxd "github.com/canonical/lxd/client"
```

Comment:
Please remove this change.

Reply:
Removed.

Reply:
By any chance have you found a way to configure this in VSCode (or any other editor) to not be added in automatically?

Reply:
I use `"go.lintTool": "golint",`

Reply:
Also:

```
"go.lintOnSave": "file",
    "go.useLanguageServer": true,
    "[go]": {
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.fixAll": true,
            "source.organizeImports": false,
        }
    },
```

Reply:
Thanks I'll give them a try!

Reply:
Looking good, setting `source.organizeImports` to `false` does the trick.